pandas/tests/indexes/test_interval.py

```diff
@@ -6,13 +6,19 @@
 from pandas import (Interval, IntervalIndex, Index, isna,
                     interval_range, Timestamp, Timedelta,
                     compat, date_range, timedelta_range, DateOffset)
+from pandas.compat import zip
 from pandas.tseries.offsets import Day
 from pandas._libs.interval import IntervalTree
 from pandas.tests.indexes.common import Base
 import pandas.util.testing as tm
 import pandas as pd
 
 
+@pytest.fixture(scope='class', params=['left', 'right', 'both', 'neither'])
+def closed(request):
+    return request.param
```

Comment:
I used this fixture to avoid having `@pytest.mark.parametrize('closed', ['left', 'right', 'both', 'neither'])` decorators everywhere.  This should be picked up by any test that has `closed` as  an argument, without needing an explicit decorator.